routes/store/order.py

```diff
@@ -11,6 +11,39 @@ def store_order_view(db_connection):
     orders = cursor.fetchall()
     return render_template('store/orders/view-orders.j2', orders_list=orders)
 
-@store_order_routes.route('/edit')
-def store_order_edit():
-    return render_template('store/orders/edit-order.j2')
+@store_order_routes.route('/edit/<int:id>', methods = ['GET', 'POST'])
+def store_order_edit(id):
+    db_connection = db.connect_to_database()
+    if request.method == 'GET':
+        order_query = 'SELECT id, customerId, status, street, city, zip, state FROM Orders WHERE id = %s;' % (id)
+        order_cursor = db.execute_query(db_connection=db_connection, query=order_query)
```

Comment:
It feels like the `% (id)` pattern is dangerous.  As opposed to passing in a third argument in `db.execute_query()` of `query_params=[id]`.  What do you think?

Reply:
Yes, forgot to follow that pattern here